app-android/src/main/res/xml/file_provider_path.xml

```diff
@@ -15,4 +15,7 @@
     <files-path
         name="files"
         path="." />
-</paths>
\ No newline at end of file
+    <root-path
```

Comment:
Why has this been added?

Reply:
The <root-path> element tells the FileProvider where to find the root directory of our app’s private storage. Without this, the FileProvider wouldn’t know where to look, and we wouldn’t be able to share the APK file.

Reply:
If that was the case, why did everything else work without it (e.g. opening submission assignment files)? I don't think it is needed. Please test removing it.

Reply:
I tested without this it's not working.

Failed to find configured root that contains / data/app/~~08Rm8hFVcAc05pWaKjKlyg==/ com.toughra.ustadmobile-08fWnSG-ZblrNg0W HQoPzQ==/base.apk